btb_phylo.py

```diff
@@ -65,18 +54,18 @@ def get_indexes_to_remove(df, parameter):
             indexes (pandas index object): indexes to remove from dataframe
     """
     indexes = pd.Index([])
-    for submission_no in df.submission.unique():
-        parameter_max = df.loc[df["submission"]==submission_no][parameter].max()
-        if len(df.loc[(df["submission"]==submission_no) & (df[parameter] == parameter_max)]) > 1:
-            warnings.warn(f"Submision {submission_no} is duplicated and has the same "
-                          f"{parameter} value\nSkipping submision {submission_no}")
+    for submission_no in df.Submission.unique():
+        parameter_max = df.loc[df["Submission"]==submission_no][parameter].max()
+        if len(df.loc[(df["Submission"]==submission_no) & (df[parameter] == parameter_max)]) > 1:
+            warnings.warn(f"Submision {submission_no} is duplicated and duplicates have "
+                          f"the same {parameter} value\nSkipping submision {submission_no}")
```

Comment:
Suggest that one is retained?

Reply:
Yes, I think you're right we should retain one. I was supprised to see at least 100 samples with the same submission number and identical `pcMapped` value. Not sure what the policy should be for choosing which one to retain, perhaps go on a second parameter? `Ncount` might be better than pcMapped for choosing samples too? Will look into this tomorrow.